packages/core-components/src/layout/HeaderActionMenu/HeaderActionMenu.test.tsx

```diff
@@ -22,7 +22,9 @@ import userEvent from '@testing-library/user-event';
 
 describe('<ComponentContextMenu />', () => {
   it('renders without any items and without exploding', async () => {
-    await renderInTestApp(<HeaderActionMenu actionItems={[]} />);
+    await expect(
+      renderInTestApp(<HeaderActionMenu actionItems={[]} />),
+    ).resolves.not.toThrow();
   });
```

Comment:
any content we can assert the existence of here? this wouldn't necessarily catch things within error boundaries